File: benchmarks/julia/prime.jl
function is_prime(n)
    if n < 2
        return false
    end
    if n == 2
        return true
    end
    if n % 2 == 0
        return false
    end
    
    for i in 3:2:isqrt(n)
        if n % i == 0
            return false
        end
    end
    return true
end

n = 999998727
println("$n is $(is_prime(n) ? "prime" : "not prime")")

File: benchmarks/php/prime.php
<?php
function isPrime($n) {
    if ($n < 2) return false;
    if ($n == 2) return true;
    if ($n % 2 == 0) return false;
    
    for ($i = 3; $i <= sqrt($n); $i += 2) {
        if ($n % $i == 0) return false;
    }
    return true;
}

$n = 999998727;
echo "$n is " . (isPrime($n) ? "prime" : "not prime") . "\n";
?>